java/server/src/org/openqa/selenium/remote/server/SessionCleaner.java

```diff
@@ -115,7 +116,8 @@ void checkExpiry() {
           try {
             deleteSession.call();
           } catch (Exception e) {
-            throw new RuntimeException(e);
+            log.log(Level.WARNING, "Could not delete session " + session.getSessionId(), e);
+            continue;
```

Comment:
I don't think it should 'continue' here, the Session will not be removed and it will perpetually try to delete this session. It should fall through, but the call to `driverSessions.deleteSession(sessionId);` will likely also need to be wrapped in a try/catch.